neurom/view/__init__.py

```diff
@@ -1,33 +0,0 @@
-# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
```

Comment:
I think we should add a deprecation warning here, pointing people to where they should look for the functions now